packages/graphiql/src/components/__tests__/GraphiQL.spec.js

```diff
@@ -143,4 +143,130 @@ describe('GraphiQL', () => {
     );
     expect(graphiQL.state().variableEditorOpen).toEqual(false);
   });
+
```

Comment:
awesome work! 1.5%+ coverage, nice

let's add a class component test too?

Reply:
👍 amended my previous commit to include a test.

Reply:
wow that was fast! LGTM now, thanks for shoring this up!